build_utils/release_tox.ini

```diff
@@ -28,23 +31,42 @@ commands = {[testenv]test_command}
 install_command = python -m pip install {opts} {packages}
 deps =
     numpy
-    -r{toxinidir}/tests_and_analysis/tox_requirements.txt
+    -r{[testenv]requirements_dir}/tox_requirements.txt
 commands_pre =
     python -m pip install \
     --force-reinstall \
     'euphonic[matplotlib,phonopy_reader,brille]=={env:EUPHONIC_VERSION}' \
     --only-binary 'euphonic'
 commands = {[testenv]test_command}
 
-[testenv:pypi-py310-min]
+[testenv:pypi-py310-min-win-linux]
 install_command = python -m pip install --force-reinstall {opts} {packages}
+platform =
+    linux: linux
+    windows: win32
 deps =
-    numpy==1.21.3
+    numpy==1.24.0
 commands_pre =
     python -m pip install --force-reinstall \
-        -r{toxinidir}/tests_and_analysis/minimum_euphonic_requirements.txt
+        -r{[testenv]requirements_dir}/minimum_euphonic_requirements.txt
```

Comment:
The one question is whether to make this a constraints rather than a requirements file. Constraints may (should?) affect the resolver of the setup-requires in the pyproject.toml meaning you don't need to pre-install numpy. 

I think though that this becomes a later problem after e.g. #332

Reply:
That does seem like a better match to what we are trying to do, yes. It still requires things to be update in two places, though. I wonder if the cleanest option is to replace the pip install with `uv install --resolution=lowest` and automatically detect the lowest supported version.

It might be worth bumping the minimum supported h5py to get rid of the platform-dependent complication...